src/main/java/uk/gov/hmcts/divorce/divorcecase/tab/CaseTypeTab.java

```diff
@@ -200,10 +200,22 @@ private void buildConditionalOrderTab(ConfigBuilder<CaseData, State, UserRole> c
         configBuilder.tab("conditionalOrder", "Conditional Order")
             .forRoles(CASE_WORKER, LEGAL_ADVISOR, SOLICITOR, SUPER_USER)
             .showCondition("coApplicant1SubmittedDate=\"*\"")
+            .label("labelConditionalOrderDetails-Applicant1",
+                "applicationType=\"jointApplication\" AND coApplicant1ApplyForConditionalOrder=\"*\"",
+                "### Applicant 1")
             .field("coApplicant1ApplyForConditionalOrder")
+            .field("coApplicant1ConfirmInformationStillCorrect")
             .field("coApplicant1SubmittedDate")
             .field("coApplicant1ChangeOrAddToApplication")
             .field("coApplicant1StatementOfTruth")
+            .label("labelConditionalOrderDetails-Applicant2",
+                "applicationType=\"jointApplication\" AND coApplicant2ApplyForConditionalOrder=\"*\"",
+                "### Applicant 2")
+            .field("coApplicant2ApplyForConditionalOrder")
+            .field("coApplicant1ConfirmInformationStillCorrect")
+            .field("coApplicant2SubmittedDate")
+            .field("coApplicant2ChangeOrAddToApplication")
+            .field("coApplicant2StatementOfTruth")
```

Comment:
do you want to display these fields only for joint ?

Reply:
@nitinprabhu yes but I thought they'd only show up if they were filled in, and they'd only be filled in in a joint application. 
I can add the condition for them if you prefer?

Just noticed coApplicant1ConfirmInformationStillCorrect should be coApplicant2ConfirmInformationStillCorrect so I'll update that

Reply:
normally they would be displayed only if they have some value. So if for sole none of the above field have values populated then it should be good otherwise we will need the condition.